src/features/call/components/TargetEvents.tsx

```diff
@@ -0,0 +1,126 @@
+import { FC, ReactNode, Suspense } from 'react';
+import {
+  Accordion,
+  AccordionDetails,
+  AccordionSummary,
+  Box,
+  Button,
+  Fade,
+  Typography,
+} from '@mui/material';
+import { ExpandMore } from '@mui/icons-material';
+
+import ZUILogoLoadingIndicator from 'zui/ZUILogoLoadingIndicator';
+import { ZetkinCallTarget } from '../types';
+import useEventCallActions from '../hooks/useEventCallActions';
+import MyActivityListItem from 'features/home/components/MyActivityListItem';
+import ZUITimeSpan from 'zui/ZUITimeSpan';
+import { removeOffset } from 'utils/dateUtils';
+import { ZetkinEvent } from 'utils/types/zetkin';
+import useAllEvents from 'features/events/hooks/useAllEvents';
+
+interface TargetEventsProps {
+  event: ZetkinEvent;
+  target: ZetkinCallTarget;
+}
+
+const TargetEvents: FC<TargetEventsProps> = ({ event, target }) => {
```

Comment:
The name of this component is plural, and for a long time I read it as being a list of events, but I'm beginning to realize that it only renders a single event and that the name is probably misleading.